### PR TITLE
Update Region Tags

### PR DESCRIPTION
**Pull request by averikitsch**
Ongoing region tag updates/removals
*Docs are frozen at last commit*

### CODE REVIEW

**Comment by ace-n** [comment]
Are the samples without any region tags not used in the docs at all? Or are you trying to include the _whole file_ instead of a specific region?

(If these samples are used somewhere in the docs, I'd argue they should still have region tags - albeit unique ones.)

**Comment by averikitsch** [comment]
Right, the removed region tags have no associated doc (and we don't want the sample tracker picking them up). I'm either updating or removing just the tag.

File: appengine/endpoints/app.js [hunk 1]
```diff
@@ -15,15 +15,12 @@
...
-// [START setup]
 const express = require('express');
 const bodyParser = require('body-parser');
 const Buffer = require('safe-buffer').Buffer;
 
 const app = express();
 app.use(bodyParser.json());
-// [END setup]
 
 app.post('/echo', (req, res) => {
```
Comment (ace-n) [inline appengine/endpoints/app.js]: Is this sample not used in the docs anymore? If so, can/should we delete it outright?

File: appengine/grunt/Gruntfile.js [hunk 2]
```diff
@@ -15,7 +15,6 @@
 
 module.exports = function (grunt) {
   grunt.initConfig({
-    // [START config]
```
Comment (ace-n) [inline appengine/grunt/Gruntfile.js]: Ditto to above - are we still using this? If not, can we delete it?

File: appengine/errorreporting/app.js [hunk 3]
```diff
@@ -13,15 +13,12 @@
  * limitations under the License.
  */
 
-// [START app]
```
Comment (ace-n) [inline appengine/errorreporting/app.js]: Ditto to above - are we still using this? If not, can we delete it?

File: appengine/metadata/standard/server.js [hunk 4]
```diff
@@ -15,7 +15,6 @@
 
 'use strict';
 
-// [START appengine_metadata]
```
Comment (ace-n) [inline appengine/metadata/standard/server.js]: Ditto to above - are we still using this? If not, can we delete it?

File: appengine/mongodb/server.js [hunk 5]
```diff
@@ -31,7 +31,6 @@ const pass = nconf.get('mongoPass');
 const host = nconf.get('mongoHost');
 const port = nconf.get('mongoPort');
 
-// [START client]
```
Comment (ace-n) [inline appengine/mongodb/server.js]: Ditto to above - are we still using this? If not, can we delete it?

File: appengine/parse-server/server.js [hunk 6]
```diff
@@ -15,7 +15,6 @@
 
 'use strict';
 
-// [START app]
```
Comment (ace-n) [inline appengine/parse-server/server.js]: Ditto to above - are we still using this? If not, can we delete it?

**Pull request merged**